### ISSUE TITLE
Freezes on quit, does not terminate

### ISSUE THREAD

**Issue by grnassar** [AntiMicroX/antimicrox#480]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Running AntiMicroX in Windows 10, running as administrator.
When I choose "quit" the program freezes, and rather than terminate its icon remains in the tray.  If I instead close via the "X" window button, the window itself closes but the icon remains in the tray.  ("Close to tray" is explicitly turned off in the settings, as is "minimize to taskbar" coincidentally.)  Opening Task Manager confirms that the executable is still running, at which point it can be forcibly closed from there.

### Expected Behavior

Program terminates on quit.

### Steps To Reproduce

Start program
Either go to the menu bar and choose "App -> Quit," or right-click on the icon in the tray and select "Quit" from the pop-up menu.

### Environment

```markdown
- OS: Windows 10 (21H2)
- AntiMicroX version: 3.2.4
```

**Comment by pktiuk** [comment]
Possibly duplicate of https://github.com/AntiMicroX/antimicrox/issues/465

Could you try to re-download and reinstall AntiMicroX installer?

**Comment by grnassar** [comment]
I did, though I didn't uninstall first. Should I uninstall/reinstall?

**Comment by grnassar** [comment]
Apologies - ignore. I didn't redownload, just reinstalled. Misunderstood
the request until I read the related but report. Thank you. I will try that
shortly.

**Comment by pktiuk** [comment]
@grnassar   
Did it help?

**Comment by AntumDeluge** [comment]
I have the same issue, but only with self-built app on Windows using MinGW-w64:

*Building AntiMicroX from master branch at 1722f6d631a61c0147f4d8cd2529fa2f8bdcd814.*

*System info:*
```
$ echo $OS
Windows_NT

$ uname -rv
3.3.5-341.x86_64 2022-07-04 21:33 UTC
```

According to #465 & #472 it is an incompatible version of SDL2.

```
$ sdl2-config --version
2.0.22
```

The fix in #472 is to downgrade SDL2 to version 2.0.20. Downgrading from 2.0.22 to 2.0.20 did indeed fix the problem.

Any clues as to what might be the conflict between *.20 & *.22?

**Comment by AntumDeluge** [comment]
The [patch was added upstream](https://github.com/libsdl-org/SDL/commit/d2c4d74dd0302369251dce9375b49f394e54f76a) but the problem does not persist in 2.24.0.
